cli/test/integration/cli/cli_test.go

```diff
@@ -295,6 +295,18 @@ func TestTerminalOutput(t *testing.T) {
 	require.Contains(t, term.Render(), "\x1b[32mINFO")
 }
 
+func TestNoPickerUsed(t *testing.T) {
```

Comment:
nit: it's a little hard to understand the name of this test out of context.

```suggestion
func TestTargetPatternFile(t *testing.T) {
```